### DISCUSSION TITLE
Does skywalking have a middleware adaptation list？

### DISCUSSION THREAD

**Discussion by Yaoz1207** [apache/skywalking#8946] [Q&A] [answered]
I'm an employee of a middleware company from China. I want to know about skywalking's compatibility certification.

**Comment by wu-sheng** [comment, marked as answer]
Every language agent has its own supported list.

**Reply by Yaoz1207** [reply]
link please

**Reply by Superskyyy** [reply]
> link please

You can find the list in every auto-instrumentation agent documentation on SkyWalking website.

**Reply by Superskyyy** [reply]
> link please

https://skywalking.apache.org/docs/ start here find each agent.

**Reply by Yaoz1207** [reply]
tanks

**Comment by wu-sheng** [comment]
Notice, this is an open source project, not your service provider.
Try to spend time on yout owm, before simply ask everything.

**Reply by Yaoz1207** [reply]
ok，thanks
